template/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.saigontechnolgy.rnbaseproject">
```

Comment:
why need remove package?

Reply:
I followed the steps on this page (https://react-native-community.github.io/upgrade-helper/?from=0.70.7&to=0.71.7) to upgrade my RN to the latest version